app/views/users/_form.html.erb

```diff
@@ -28,9 +28,11 @@
   <%= f.label :zipcode %>
   <%= f.text_field :zipcode %>
   <br>
-
-  <%= f.label :username %>
-  <%= f.text_field :username  %>
+  
+  <% if @user != current_user %>
```

Comment:
@martensonbj couldn't we also do <% unless current_user %> to just check if someone is signed in? You can't get to this page unless you're the current user anyway.